demisto_sdk/commands/common/tools.py

```diff
@@ -2766,3 +2766,66 @@ def remove_copy_and_dev_suffixes_from_str(field_name: str) -> str:
             if field_name.endswith(suffix):
                 field_name = field_name[:-len(suffix)]
     return field_name
+
+
+def get_invalid_incident_fields_from_mapper(
+    mapper_incident_fields: dict, mapping_type: str, content_fields: list
```

Comment:
```suggestion
    mapper_incident_fields: Dict, mapping_type: str, content_fields: List[str]
```
Its better to use types from typing where possible, and give subtypes where we know them

Reply:
done it for all functions